assets/xml/overlays/ovl_En_Holl.xml

```diff
@@ -0,0 +1,9 @@
+<Root>
+    <File Name="ovl_En_Holl" BaseAddress="0x80899960" RangeStart="0x0BF0" RangeEnd="0x0C58" Segment="128">
+        <Array Name="D_8089A550" Count="4" Offset="0x0BF0">
+            <Vtx/>
+        </Array>
+
+        <DList Name="D_8089A590" Offset="0x0C30"/>
```

Comment:
You can name those if you want. Don't forget to rename them in yours actor's code
```suggestion
        <Array Name="gHollPlaneVtx" Count="4" Offset="0x0BF0">
            <Vtx/>
        </Array>

        <DList Name="gHollPlaneDL" Offset="0x0C30"/>
```

Reply:
Shouldn't it begin with an "s" instead of a "g"?

Reply:
We usually use `s` for static variables. Since those variables aren't extracted as static, I think `g` should be used.

Reply:
they were static before I extracted them, when they were just sitting in my .c file though, so I'm confused. Isn't g for global? These aren't global are they?

Reply:
I did some research, and apparently, the difference between global and static is just that you can extern globals in other files, but not statics. Since this is externed in the extracted asset .h file, it does indeed appear to be global. Thanks!

Reply:
Fixed in https://github.com/zeldaret/mm/pull/270/commits/db7632c5beb000bf4d8322b850ad6c1f4858abd1